#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_fix_no_medium_found_error.dpatch <mika@grml.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix the "cannot open /dev/scd0: No medium found" issue

@DPATCH@

--- a/scripts/live-helpers
+++ b/scripts/live-helpers
@@ -67,24 +67,6 @@ is_supported_fs ()
 
 get_fstype ()
 {
-	local FSTYPE
-	local FSSIZE
-
-	# fstype misreports LUKS devices
-	if is_luks "${1}"
-	then
-	    /lib/udev/vol_id -t ${1} 2>/dev/null
-	    return
-	fi
-
-	eval $(fstype < ${1} 2>/dev/null)
-
-	if [ "${FSTYPE}" != "unknown" ]
-	then
-		echo ${FSTYPE}
-		return 0
-	fi
-
 	/lib/udev/vol_id -t ${1} 2>/dev/null
 }
 
